### DISCUSSION TITLE
Bug: i am getting one error while trying to play music

### DISCUSSION THREAD

**Discussion by ummaabhignakumarreddy** [eritislami/evobot#504] [Q&A]
I am getting error like this 

 The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.

help me by dm me Jaguar || Alone Everytime ||#1111   plzzz help meeeeee

**Comment by mihawkman** [comment]
replace your old youtube api key to a new one
